model/competencyRequest.ts

```diff
@@ -0,0 +1,5 @@
+export interface CompetencyRequest  {
```

Comment:
Could this be called competency response?

```suggestion
export interface CompetencyResponse  {
```